bundle/src/main/java/com/jfrog/ide/eclipse/configuration/XrayServerConfigImpl.java

```diff
@@ -37,7 +47,25 @@ public String getPassword() {
 	}
 
 	private String getValue(String key) {
-		String str = service.getString(PreferenceConstants.XRAY_QUALIFIER, key, "", null);
-		return StringUtils.trim(str);
+		return trim(service.getString(PreferenceConstants.XRAY_QUALIFIER, key, "", null));
+	}
+
+	@Override
+	public ProxyConfig getProxyConfig(String hostUrl) {
```

Comment:
I suggest renaming this method name to *getProxyConfForTargetUrl*.
I also suggest adding the following comment:
*This methods reads the http proxy configuration set in Eclipse configuration and returns the proxy config for the Xray URL, which is sent as an argument.*

Reply:
I added this comment in the interface